pkg/scheduler/algorithm/predicates/metadata_test.go

```diff
@@ -525,3 +525,224 @@ func TestPredicateMetadata_ShallowCopy(t *testing.T) {
 		t.Errorf("Copy is not equal to source!")
 	}
 }
+
+// TestGetTPMapMatchingIncomingAffinityAntiAffinity_AntiAffinity tests against method getTPMapMatchingIncomingAffinityAntiAffinity
+// on Anti Affinity cases
+func TestGetTPMapMatchingIncomingAffinityAntiAffinity_AntiAffinity(t *testing.T) {
```

Comment:
The "_AntiAffinity" suffix seems to be unintentional. Please remove.

Reply:
Done.